run_sync.py

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
This file should be removed, I'm not sure what it's for. Running `tach sync` is done through the `cli.py` file based on command-line arguments.